src/web/modules/layout/user_banner.njk

```diff
@@ -0,0 +1,15 @@
+<div class="user-profile">
+  <div class="user-profile__container">
+    <div class="user-profile__item">
+      <img class="avatar" height="20" width="20" src="{{ user.avatarUrl }}">
```

Comment:
How does it render when user doesn't have avatar. (or will user always have an avatar?)

Reply:
GitHub generates a user avatar if no picture is provided so it should always respond with a valid URL -- if this isn't the case for some reason it will render the image not found logo (this differs browser to browse). Because there is an explicit width and height the flow of the page shouldn't be impacted